core/modules/layout/layout.module

```diff
@@ -709,6 +718,15 @@ function layout_contextual_links_view_alter(&$element, $items) {
       $links['layout-configure-redirect-']['fragment'] = 'configure-block:' . $uuid;
     }
   }
+  if (isset($links['layout-contextual-remove-'])) {
+    backdrop_add_library('system', 'backdrop.ajax');
+    backdrop_add_library('system', 'ui.dialog');
+    $links['layout-contextual-remove-']['attributes'] = array(
+        'class' => array('use-ajax'),
+        'data-dialog' => TRUE,
+        'data-dialog-options' => json_encode(array('dialogClass' => 'front-dialog', 'width' => '600')),
```

Comment:
Two extra spaces in indentation here.

Rather than using `backdrop_add_library()`, we should probably append to $element['#attached'] as:

```
$element['#attached']['library'][] = array('system', 'backdrop.dialog.ajax');
```

The single library `'backdrop.dialog.ajax'` should also include all the necessary files without separate libraries.